tasks/go_test.py

```diff
@@ -1203,6 +1208,14 @@ def get_modified_packages(ctx) -> List[GoModule]:
         else:
             modules_to_test[best_module_path] = GoModule(best_module_path, targets=[relative_target])
 
+    # Clean up duplicated paths to reduce Go test cmd length
+    for module in modules_to_test:
+        modules_to_test[module].targets = clean_nested_paths(modules_to_test[module].targets)
+        if (
+            len(modules_to_test[module].targets) >= 150
```

Comment:
nitpick
```suggestion
            len(modules_to_test[module].targets) >= WINDOWS_MAX_CLI_LENGTH
```
And define this magic number somewhere